dev/docker/ballista.dockerfile

```
@@ -49,7 +49,9 @@ ARG PROTOC_VERSION=21.4
 RUN mkdir /tmp/protoc
 WORKDIR /tmp/protoc
 
-RUN export PROTO_ZIP="protoc-${PROTOC_VERSION}-linux-x86_64.zip" && \
+RUN CPU_ARCH=`uname -m` && \
+  if [ $CPU_ARCH=="aarch64" ]; then CPU_ARCH="aarch_64"; fi && \
```

```suggestion
if [ [ $CPU_ARCH == "aarch64" ] || [ $CPU_ARCH == "amd64" ] ] ; then CPU_ARCH="aarch_64"; fi && \
```

On my Mac M1, `uname -m` returns `arm64` so I had to make the change above.